xapian-applications/omega/omindexcheck.cc

```diff
@@ -130,6 +132,14 @@ index_test()
 		  {"Zhead", "Zhello", "Zopenoffic", "Zimpress"}});
     tests.insert({"sof/libarchive_openoffice_presentation_template.sti",
 		  {"ZSproject", "ZSresearch", "Zhead", "Ztext"}});
+
+    // OpenXML formats
```

Comment:
I think we should above calling these formats "OpenXML", and just say "OOXML" instead.  Microsoft subverted the ISO standardisation process (for some history see https://en.wikipedia.org/wiki/Office_Open_XML#Standardization_process).

It's not really an "Open" format - sure the specification is public, but the format was just submitted fully formed by Microsoft rather than developed in an open way.  It's really no more "Open" than the older OLE-stream Microsoft formats they replaced with it (which are also now publicly documented, but only after Microsoft lost an anti-trust case).

The format is sadly widely used so it's unhelpful to refuse to support it, but we should avoid being sucked into their PR games.

Reply:
Thank you for the insight. I have made the necessary changes.